### PR TITLE
VIH-9613 display work hours clash

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-9613

### Change description ###

* Display an icon when there is a work hours clashing for an allocated CSO
* Display a pending change icon when an allocation has been updated but not yet confirmed
* Add a tooltip directive to show on applied elements

image

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/dropdown-menu/justice-user-menu-stub.component.ts [hunk 1]
```diff
@@ -14,4 +14,7 @@ export class JusticeUserMenuStubComponent {
     clear() {
         console.log('stub method');
     }
+    enabled(enabled: boolean) {
+        console.log(`Justice User Menu component enabled state changed to : ${enabled}`);
```
Comment (marcogagliardi) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/dropdown-menu/justice-user-menu-stub.component.ts:18]: is it possible to use the logger?

**Review by marcogagliardi — COMMENTED** [review]
Looks really good. Just an overlooked log otherwise really good.

File: AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/dropdown-menu/justice-user-menu-stub.component.ts (hunk 1 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/dropdown-menu/justice-user-menu-stub.component.ts:18]: this is a stub component for testing, not the real one.

Comment (marcogagliardi) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/dropdown-menu/justice-user-menu-stub.component.ts:18]: ok no worries.

**Review by marcogagliardi — APPROVED** [review]
👌🏽

**Pull request merged**